app/src/components/posts/PostListItem.vue

```diff
@@ -0,0 +1,228 @@
+            <span v-if="post.post_struct.location.country !== 'Deutschland'">
+              {{ post.post_struct.location.country }}
+            </span>
+            <em
+              v-if="
+                distance && post.post_struct.location.country === 'Deutschland'
+              "
+            >
+              (in {{ distance }})
+            </em>
+          </strong>
+          &mdash;
+          <span v-html="post.task" />
+        </v-list-item-subtitle>
+      </v-list-item-content>
+    </v-list-item>
+    <!-- Detail Card -->
+    <v-card
+      v-if="active && showDetail"
+      class="details-smartphone"
+      :class="{
+        'details-smartphone-visible': active,
+        'details-smartphone-hidden': !active,
+      }"
+    >
+      <v-card-text>
+        <template v-for="el in postTable">
+          <div v-if="post[el.key]" :key="el.key">
+            <h3>{{ el.label }}</h3>
+            <p v-if="el.link">
+              <a :href="post[el.link]" target="_blank">{{ post[el.key] }}</a>
+            </p>
+            <p v-else v-html="post[el.key]" />
+          </div>
+        </template>
+      </v-card-text>
+      <v-card-actions class="actions">
+        <div class="tags">
+          <template v-for="(category, i) in post.categories">
+            <v-chip :key="i" class="mr-2 mt-2">{{ category }}</v-chip>
+          </template>
+        </div>
+        <v-spacer></v-spacer>
+        <div class="my-2 mt-6">
+          <v-btn
+            class="my-2"
+            dark
+            large
+            color="#054C66"
+            :href="post.link"
+            target="_blank"
+          >
+            Zum Angebot
+          </v-btn>
+        </div>
+      </v-card-actions>
+    </v-card>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import Post from "@/models/post";
+import Location from "@/models/location";
+
+export default Vue.extend({
+  name: "PostListItem",
+  props: {
+    post: {
+      type: Object as () => Post,
+      required: true,
+    },
+    active: {
+      type: Boolean,
+      default: false,
+    },
+    showDetail: {
+      type: Boolean,
+      default: false,
+    },
+    location: {
+      type: Object as () => Location,
+      default: undefined,
+    },
+  },
+  data: function () {
+    return {
+      postTable: [
+        { key: "location", label: "Einsatzort" },
+        { key: "task", label: "Aufgabe" },
+        { key: "contact", label: "Ansprechpartner" },
+        { key: "organization", label: "Organisation" },
+        { key: "target_group", label: "Zielgruppe" },
+        { key: "timing", label: "Einstiegsdatum / Beginn" },
+        { key: "effort", label: "Zeitaufwand" },
+        { key: "opportunities", label: "Möglichkeiten" },
+        { key: "prerequisites", label: "Anforderungen" },
+        { key: "language_skills", label: "Sprachen" },
+        { key: "source", label: "Quelle", link: "link" },
+      ],
+    };
```

Comment:
We should find a more descriptive name than postTable.

Reply:
I refactor it to 'columns'. Also there could be more refactoring to combine duplicated code from PostCard.vue and Post List Item.vue.

PostListItem is used for the left side of the posts page. Here the detail card for a selected post is only shown on mobile. However, on dekstop the PostListItem component is used for showing the details for the selected post instead. 

If this pr will be merged a new issue could be opened for this.